src/main/resources/nwoolcan/view/dashboard.fxml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
```

Comment:
Warning: are you sure you want to include all the subpackages under `control` and `layout`? I would avoid them